python_modules/libraries/dagster-teradata/dagster_teradata/teradata_io_manager.py

```diff
@@ -0,0 +1,135 @@
+from abc import abstractmethod
```

Comment:
What is role of this class

Reply:
We don't need "teradata_io_manager.py". I will remove that file completely